.travis.yml

```diff
@@ -1,10 +1,6 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
```

Comment:
why you remove other PHP versions of travis?

Reply:
symfony 4.0 requires PHP 7.1.3 as minimum.

Reply:
See my first comment in this PR.